device_plugins/deploy_qat.md

```diff
@@ -1,4 +1,19 @@
 # Create Intel QAT Device Plugin CR
+Intel® QuickAssist Technology (Intel® QAT) provides cryptographic and compression acceleration capabilities used to improve performance and efficiency across the data center.
+For more details see https://intel.github.io/quickassist
+
+## Prerequisite
+Before deploying the QAT plugin, make sure iommu MCO is deployed and iommu is enabled.
+By default, half of the virtual functions are used for compression and the other half is used for cryptography. This default behaviour can be customised via a config map which configures the hardware using sysfs. The config values could be `asym`, `sym` and `dc`, for more details see [cfg_services](https://github.com/torvalds/linux/blob/42e66b1cc3a070671001f8a1e933a80818a192bf/Documentation/ABI/testing/sysfs-driver-qat)
+
+To create a configmap, run the following command:
+```
+oc create configmap --namespace=openshift-operators --from-literal "qat.conf=ServicesEnabled=<option>" qat-config
+```
+The option can be any combination of values described above.
+When using the operator for deploying the plugin with provisioning config, use provisioningConfig field for the name of the ConfigMap, then the config is passed to initcontainer through the volume mount.
+
+There's also a possibility for a node specific congfiguration through passing a nodename via NODE_NAME into initcontainer's environment and passing a node specific profile (qat-$NODE_NAME.conf) via ConfigMap volume mount.
```

Comment:
I am thinking about whether we should expose the configmap detail to the OCP customers.

Reply:
It's totally fine to have it. The default setting without the configmap is also fine but if the cluster admin wants to opt-in for a custom per-node configuration, they can do so with the configmap and by setting the name of the confimap in `QatDevicePlugin` custom resource.

Reply:
I'd like to put it into some kind of Advanced Configuration Section in the readme.